src/components/cluster-common.js

```diff
@@ -96,3 +96,16 @@ export const PeriodicCookieSetter = ({ namespace, runtimeName, leading }) => {
     { ms: 5 * 60 * 1000, leading })
   return null
 }
+
+export const SaveFilesHelp = () => {
+  return h(Fragment, [
+    p([
+      'If you want to save some files permanently, such as input data, analysis outputs, or installed packages, ',
+      h(Link, {
+        href: 'https://support.terra.bio/hc/en-us/articles/360026639112',
+        ...Utils.newTabLinkProps
+      }, ['move them to the workspace bucket.'])
+    ]),
+    p(['Note: Jupyter notebooks are autosaved to the workspace bucket, and deleting your disk will not delete your notebooks.'])
```

Comment:
Why does this say Jupyter notebooks specifically?
1. If it is actually only about Jupyter notebooks I wonder if it is appropriate to have in this cluster-common file 
2. Or if it applies to all cloud environments then this wording should be updated

Reply:
At the moment, I believe this is specific to Jupyter. My understanding is that RStudio does not have "notebook" documents the same way Jupyter does. I do not know the file/data management model for Galaxy.